packages/cli/src/alpha.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CliInitializer } from './wiring/CliInitializer';
+
```

Comment:
Let's add a very clear `console.log` that this is an experimental entry point for internal development of the CLI that should not be used, just to avoid any kind of confusion about what this entry point is for. As opposed to other experimental entry points I'd also like us to avoid shipping any new features here so that we don't tempt anyone to use this. As soon as we're able to mirror the existing CLI using the new structure we should get rid of this entry point too.